src/mower_logic/src/mower_logic/mower_logic.cpp

```diff
@@ -224,19 +224,19 @@ bool setGPS(bool enabled) {
 }
 
 
-/// @brief If the BLADE Motor is not in the requested status (enabled),we call the 
+/// @brief If the BLADE Motor is not in the requested status (enabled),we call the
```

Comment:
What's with the whitespace changes? Removing windows line endings?

Reply:
My editor is configured to trim whitespace at the end of the lines automatically when saving. It seems that this file had some trailing whitespace which got removed. Ideally we should put a source formatter configuration to this repo and then a github action that checks formatting, so all the devs are following the same formatting rules.